gems/quilt_rails/README.md

```diff
@@ -829,10 +834,28 @@ The `Performance::Reportable` mixin uses [https://github.com/Shopify/statsd-inst
 
 ### Generators
 
-#### `quilt:install`
+#### `quilt_rails:install`
+
+Run both generator below.
+
+##### `quilt_rails:sewing_kit:install`
+
+Adds a basic `sewing-kit.config.ts` file and basic linting, format configuration.
+This generator assume the use of [`sewing-kit`](https://github.com/Shopify/sewing-kit)
```

Comment:
IMO, we should use `@shopify/sewing-kit` over `sewing-kit`. It helps differentiate between the ruby vs node libs a bit better too.